tools/packer/setup.sh

```diff
@@ -55,3 +58,11 @@ done
 # Enable services
 systemctl daemon-reload
 systemctl enable lotus-daemon
+
+# Setup firewall
+ufw enable
+ufw default deny incoming
+ufw default allow outgoing
+ufw allow ssh
+ufw allow 1234   #lotus api
```

Comment:
Making the api open to the internet by default (or at all) is a pretty bad idea

Reply:
Thanks, I'll remove this rule.